db/singleton.js

```diff
@@ -0,0 +1,27 @@
+const { Pool } = require("pg");
+const path = require('path');
+require('dotenv').config({path: path.resolve(__dirname, '../.env')});
+
+class Singleton {
```

Comment:
The name is 10/10)
It is actually should be named `PostgressPoolConnection` or smth like this.

Reply:
?